codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -744,7 +744,8 @@ private void notifyWritabilityChanged(FlowState state) {
             try {
                 listener.writabilityChanged(state.stream);
             } catch (Throwable cause) {
-                logger.error("Caught Throwable from listener.writabilityChanged", cause);
+                logger.error("{} Caught Throwable from listener.writabilityChanged for {}",
+                        ctx.channel(), state.stream, cause);
```

Comment:
There are a few similar logs in `DefaultHttp2Connection` that I was not able to align because there is no references to the channel. Any ideas what we can do there?

Reply:
will need to dig a bit... That said we might be out of luck then